src/routes/index.jsx

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-import Header from '../components/UI/Header';
+
+import Layout from '../hoc/layout/Layout';
```

Comment:
Do you think `hoc` could get a more descriptive name?
Or a short doc in the module's index file?

Reply:
the high order component naming for the layout seems better since its the one to lay the header and footer over selected pages.